peps/pep-9999.rst

```diff
@@ -105,27 +115,83 @@ interval math or numerical integration.
 Other proposed names include ``ntheory`` (like SymPy's submodule),
 ``integermath`` and ``imaths``.
 
+Module scope and possible library members
+=========================================
+
+This PEP proposes a new module with only the six existing functions listed
+above.  However, in order to support the module's possible future utility, and
+to illustrate its scope, we provide an exploratory list of functions that could
+one day find a home in the ``imath`` library:
```

Comment:
Sorry, but you again miss my point.

The purpose of this section is to show what can be included eventually to the module and what can't, not classification of the `factorial` as part of the Combinatorics or whatever else.

Reply:
> classification of the factorial as part of the Combinatorics or whatever else.

But I'm suggesting that's the point?  Let's go point-by-point:

Do you agree that this section should:

* show the module's possible future utility (that there are a lot of functions that could find a home here)?
* illustrate the module's scope (what kinds of things would best be in `imath` and not, say, `imath`?

Also, you wantt his section to show "what can be included eventually".  What do you mean by this exactly?  Is it one of the two things above, or something else?